packages/graphql-toolbox/src/modules/Login/Login.tsx

```diff
@@ -62,7 +62,7 @@ export const Login = () => {
         <div data-test-login-form className="grid place-items-center h-screen n-bg-neutral-90">
             <div className="w-login flex flex-col align-center justify-center bg-white shadow-md rounded p-8">
                 <div className="mb-6 text-center">
-                    <img src={Icon} alt="Neo4j Logo" className="h-12 w-12 mx-auto" />
+                    <img src={Icon} alt="Neo4j Logo" className="h-12 w-12 mb-3 mx-auto" />
```

Comment:
I'm not sure how I feel about there being zero Neo4j branding on the login page - what do other tools do?

Reply:
Yes good point, I had mixed feeling for this particular change. I quickly checked a few other products:

https://browser.neo4j.io/ - No logo in the login modal
https://bloom.neo4j.io/ - No logo in the login card (Neo4j logo on the bottom left)
https://data-importer.neo4j.io/ - Neo4j logo in the login modal

I'll get the Neo4j logo back for our login modal